releasenotes/notes/kms-grants-fix-9cdb923a9157df6e.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    **[KMS]** Fix reading of ``issuing_principal`` in ``resource/opentelekomcloud_kms_grant_v1`` (`#1593 <https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/1593>`_)
```

Comment:
Fix of https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/issues/1591 is not mentioned

Reply:
done